src/types/tests/schema-test.ts

```diff
@@ -628,6 +628,108 @@ describe('schema', () => {
     assert.deepEqual(intersection.fields, schema3.fields);
     assert.deepEqual(intersection.refinement, schema3.refinement);
   }));
+  it('tests schema union, with inline entities', async () => {
+    const manifest = await Manifest.load('./AB.schema',
+      new Loader(null, {
+        './AB.schema': `
+            schema A
+              foo: inline Foo {a: Text, b: Text}
+
+            schema B
+              foo: inline Foo {a: Text}
+            `
+      }));
```

Comment:
The following should work?

```suggestion
    const manifest = await Manifest.parse(`
            schema A
              foo: inline Foo {a: Text, b: Text}

            schema B
              foo: inline Foo {a: Text}
            `);
```

Reply:
Thanks, of course!